meter/tibber-pulse.go

```diff
@@ -163,6 +161,8 @@ func NewTibberFromConfig(ctx context.Context, other map[string]interface{}) (api
 		}
 	}()
 
+	log.INFO.Printf("Tibber pulse: User-Agent set to %s", getUserAgent())
```

Comment:
```suggestion
	log.INFO.Printf("!! User-Agent set to %s", getUserAgent())
```

Reply:
Could you please commit? We'll use !! for temporary logging to be removed later. The `tibber` prefix is already set by the parent logger.